src/components/dashboard/dashboard.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { DashboardOptions } from './dashboard.component';
+import { DashboardModule } from './dashboard.module';
+
+
+@Component({
+    selector: 'app-ux-dashboard',
+    template: `<ux-dashboard [options]="options"></ux-dashboard>`
+})
+export class DashboardComponent {
+
+    options: DashboardOptions  = {
+        columns: 3,
+        padding: 10,
+        rowHeight: 300,
+        emptyRow: false,
+        minWidth: 187
+    };
+}
+
+describe('Dashboard', () => {
+    let component: DashboardComponent;
+    let fixture: ComponentFixture<DashboardComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            imports: [DashboardModule],
+            declarations: [DashboardComponent]
+        })
+            .compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(DashboardComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should initialise correctly', () => {
+        expect(component).toBeTruthy();
+    });
+
+    it('should change columns option if columns option adjusted', async() => {
+        expect(component.options.columns).toBe(3);
+
+        component.options.columns = 6;
```

Comment:
The dashboard component is `OnPush` so I'm not even sure this is triggering change detection.